src/main/scala/com/netflix/edda/mongo/MongoElector.scala

```diff
@@ -85,7 +85,7 @@ class MongoElector extends Elector {
             "ltime" -> null,
             "data" -> Map("instance" -> instance, "id" -> "leader", "type" -> "leader"))))
       // if we got an error then uniqueness failed (someone else beat us to it)
-      isLeader = if (wr.getN() == 0) true else false
+      isLeader = if (wr.getN() == 1) true else false
```

Comment:
It's not from getLastError, it's from the the writeResult and in that context getN returns the number of affected records.

https://api.mongodb.com/java/current/com/mongodb/WriteResult.html